.github/workflows/notebooks.yml

```diff
@@ -10,33 +10,29 @@ jobs:
     notebooks:
         strategy:
             fail-fast: true
-            matrix:
-                python-version: ['3.10']
-                poetry-version: ['1.4.1']
-                os: [ubuntu-latest]
-        runs-on: ${{ matrix.os }}
```

Comment:
Taking the opportunity to simplify the action, we don't actually need a matrix here.